admin/components/ui/custom/expandableSection.tsx

```diff
@@ -1,17 +1,36 @@
-import { useState } from "react";
+import { useEffect, useState } from "react";
 
 const ExpandableSection = ({
+  name,
   title,
   description,
-  open,
   children,
 }: {
+  name: string;
   title: string;
   description?: string;
-  open: boolean;
+  open?: boolean;
   children: React.ReactNode;
 }) => {
-  const [isOpen, setIsOpen] = useState(open ?? true);
+  const [isOpen, setIsOpen] = useState<boolean | null>(null);
+
+  // Load the state from local storage when the component mounts
+  useEffect(() => {
+    const savedState = localStorage.getItem("expandable-section-" + name);
+    if (savedState !== null) {
+      setIsOpen(savedState === "open");
+    }
```

Comment:
Still pass a default ?

Reply:
Great catch! I just update to allow pass as default